### PR TITLE
feat: add component for iframe

### LINKED ISSUE #39: Add component for iframe

### PR DESCRIPTION
**Pull request by askmrsinh**
closes https://github.com/wxsd-sales/roomos-device-widgets/issues/39

### CODE REVIEW

**Comment by askmrsinh** [comment]
Added optional field for Iframe URL in demo form:
localhost_5173_demos_save_id=1c565cb7-d1ca-417d-b3b9-ea94921c6390

Iframe component showing OpenStreetMap:
localhost_5173__activationId=11744ceb-6d87-49b6-a785-a4c86a9c6d2d

File: src/routes/demos/activate/.Step3DemoUuid.svelte [hunk 1]
```diff
@@ -79,7 +83,7 @@
       </div>
     </div>
     <div class="help">
-      <p>To identify a demo, take not of its ID or name</p>
+      <p>To identify a demo, take note of its ID or name</p>
```
Comment (akoushke) [inline src/routes/demos/activate/.Step3DemoUuid.svelte]: ``` it's ``` 🙈

File: src/routes/demos/activate/.Step4GoogleIntegration.svelte [hunk 2]
```diff
@@ -27,8 +27,9 @@
   <div class="column is-full content mb-0">
     <p>
       This is an advanced use case which requires setup of a Google Cloud Service Account with "Domain-wide delegation
-      of authority" to access the user’s Workspace calendar. Consult the WXSD team for help/instructions if you want to
-      try this.
+      of authority" to access the user’s Workspace calendar. Consult the <a
+        href="mailto:wxsd@external.cisco.com?subject=RoomOS%20Device%20Widgets%20(Google%20Integration)">WXSD team</a
```
Comment (akoushke) [inline src/routes/demos/activate/.Step4GoogleIntegration.svelte]: why dont you const this?

File: src/routes/demos/activate/.Step5MicrosoftIntegration.svelte [hunk 3]
```diff
@@ -29,7 +29,9 @@
   <div class="column is-full content mb-0">
     <p>
       This is an advanced use case which requires setup of a Microsoft Azure App Registration with Certificates to
-      access the user’s Office 365 calendar. Consult the WXSD team for help/instructions if you want to try this.
+      access the user’s Office 365 calendar. Consult the <a
+        href="mailto:wxsd@external.cisco.com?subject=RoomOS%20Device%20Widgets%20(Microsoft%20Integration)">WXSD team</a
```
Comment (akoushke) [inline src/routes/demos/activate/.Step5MicrosoftIntegration.svelte]: ^

File: src/routes/demos/activate/[uuid].ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import type { RequestEvent } from '@sveltejs/kit';
+import { Activation } from '../../../database/entities';
+import { LoadStrategy } from '@mikro-orm/core';
+
+export const DELETE = async (requestEvent: RequestEvent) => {
+  const activationId = requestEvent.params.uuid;
+  const db = requestEvent.locals.db;
+  const session = requestEvent.locals.session;
+
+  if (db && session && session.user && session.user.uuid) {
```
Comment (akoushke) [inline src/routes/demos/activate/[uuid].ts]:
```js
if (db && session?.user?.uuid)

**Review by akoushke — COMMENTED** [review]
being nit-picky

File: src/routes/demos/activate/[uuid].ts (hunk 4 above)
Comment (askmrsinh) [reply src/routes/demos/activate/[uuid].ts]: Done.

File: src/routes/demos/activate/.Step5MicrosoftIntegration.svelte (hunk 3 above)
Comment (askmrsinh) [reply src/routes/demos/activate/.Step5MicrosoftIntegration.svelte]: Done.

File: src/routes/demos/activate/.Step4GoogleIntegration.svelte (hunk 2 above)
Comment (askmrsinh) [reply src/routes/demos/activate/.Step4GoogleIntegration.svelte]: Done.

File: src/routes/demos/activate/.Step3DemoUuid.svelte (hunk 1 above)
Comment (askmrsinh) [reply src/routes/demos/activate/.Step3DemoUuid.svelte]: Nice catch, thanks!

**Pull request merged**